Apps/Memos/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+name: memos
+services:
+  lidarr:
+    environment:
+      PGID: "1000"
+      PUID: "1000"
+      TZ: $TZ
+    image: neosmemo/memos:0.12.2
+    deploy:
+      resources:
+        reservations:
+          memory: 64M
+    network_mode: bridge
+    ports:
+      - target: 5230
+        published: "5230"
+        protocol: tcp
+    restart: unless-stopped
+    volumes:
+      - type: bind
+        source: /DATA/AppData/memos/memos
+        target: /var/opt/memos
+    x-casaos:
+      envs:
+        - container: PUID
+          description:
+            en_us: ""
+        - container: PGID
+          description:
+            en_us: ""
+        - container: TZ
+          description:
+            en_us: ""
+      ports:
+        - container: "5230"
+          description:
+            en_us: WebUI HTTP Port
+            zh_cn: WebUI HTTP 端口
+
+      volumes:
+        - container: /config
+          description:
+            en_us: ""
+x-casaos:
+  architectures:
+    - amd64
+    - arm
+    - arm64
+  main: memos
```

Comment:
The only `service` under `services` is `lidarr`. Either `main` has to be `lidarr` or change `lidarr` to `memos`.

Reply:
I was too careless🥺 I have corrected it